WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -108,7 +118,45 @@ public static void addNewBlogPostOrPage(Context context, Blog blog, boolean isPa
         context.startActivity(intent);
     }
 
+    public static void editBlogPostorPage(Activity activity, long postOrPageId, boolean isPage) {
+        Intent intent = new Intent(activity.getApplicationContext(), EditPostActivity.class);
+        intent.putExtra(EditPostActivity.EXTRA_POSTID, postOrPageId);
+        intent.putExtra(EditPostActivity.EXTRA_IS_PAGE, isPage);
+        activity.startActivityForResult(intent, RequestCodes.EDIT_POST);
+    }
+
     public static void addMedia(Context context, Blog blog) {
         // TODO: https://github.com/wordpress-mobile/WordPress-Android/issues/2394
     }
+
+    public static void viewSSLCerts(Context context) {
+        try {
+            Intent intent = new Intent(context, SSLCertsViewActivity.class);
+            SelfSignedSSLCertsManager selfSignedSSLCertsManager = SelfSignedSSLCertsManager.getInstance(context);
+            String lastFailureChainDescription =
+                    selfSignedSSLCertsManager.getLastFailureChainDescription().replaceAll("\n", "<br/>");
+            intent.putExtra(SSLCertsViewActivity.CERT_DETAILS_KEYS, lastFailureChainDescription);
+            context.startActivity(intent);
+        } catch (GeneralSecurityException e) {
+            AppLog.e(AppLog.T.API, e);
+        } catch (IOException e) {
+            AppLog.e(AppLog.T.API, e);
+        }
+    }
+
+    public static void viewSettings(Activity activity) {
```

Comment:
Let's add `ForResult` to the names of routines that call `startActivityForResult` (ex: `viewSettingsForResult`) to make them more self-documenting.